misc/config_definitions.json

```diff
@@ -1188,7 +1188,7 @@
             "order": 0,
             "type": "boolean"
         },
-        "distributed_poller_group": {
```

Comment:
distributed_poller_group should still exist, but not be exposed to the ui since you would want to set it per poller and settings in the DB apply to all pollers.

Reply:
you're right, it should still exist but not in GUI (for first)